lib/data_migrations/merge_ifp_with_cis.rb

```diff
@@ -0,0 +1,65 @@
+class MergeIfpWithCis
+  def run_migration
+    remove_old_body_selections(adviser_ids_where_both_bodies_selected)
+    update_old_body_selections(adviser_ids_where_only_one_body_selected)
```

Comment:
Is it worth printing some details from these mutations? Might be useful for debugging purposes as we cannot have a "migrate down" sort of thing.

Reply:
Yeah, I think you're right. On a related note, I've just realised these operations are going to run on 3 and  ~160 records, respectively. Completely over estimated the target dataset size. Given the low numbers, I'm going to replace some of this SQL with simple iteration and make the updates via the Adviser model. Gives an opportunity to add some logging about the data change.